README.md

```diff
@@ -64,4 +64,12 @@ gem install bundler wdm tzinfo-data
 gem update listen middleman
 ```
 
-Once Middleman comes up, you'll be prompted by Windows Firewall. Click "Allow access" and you'll be in business!
+**NOTE**: if you get an error like this:
```

Comment:
Stylistically, @trek and I prefer to avoid "NOTE"s. The reason is that it's easy to inject a note anywhere without having to take into account the narrative flow. Because this documentation is edited collaboratively by many people over many years, it becomes very easy for the documentation to become a torrent of "NOTE"s strung together without any real rhyme or reason. It's better to refactor the note to be part of the narrative flow. In this case, I think the paragraph stands without the "NOTE:" prefix.